lumigator/frontend/src/services/datasets/datasetsService.ts

```diff
@@ -70,7 +83,11 @@ async function downloadDataset(id) {
     });
     return fileResponse.data;
   } catch (error) {
-    console.error('Error downloading dataset: ', error.message || error);
+    if (error instanceof Error) {
+      console.error('Error downloading dataset: ', error.message || error);
+    } else if (error instanceof AxiosError) {
+      console.error('Error downloading dataset: ', error.response?.data || error);
+    }
```

Comment:
here and in similar cases is also a future refactoring opportunities to properly bubble up the errors to the components or any consumers so they can handle them and get the real error message instead of silently swallowing and logging it here